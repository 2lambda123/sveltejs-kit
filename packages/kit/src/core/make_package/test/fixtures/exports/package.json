{
	"name": "exports",
	"version": "1.0.0",
	"description": "package-exports-test",
	"exports": {
		"./package.json": "./package.json",
		".": {
			"import": "./index.js"
		}
	}
}
