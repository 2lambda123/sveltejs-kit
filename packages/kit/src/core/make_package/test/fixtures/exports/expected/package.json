{
	"name": "javascript",
	"version": "1.0.0",
	"description": "package-javascript-test",
	"exports": {
		"./package.json": "./package.json",
		".": {
			"import": "./index.js"
		}
	},
	"type": "module"
}
